go.mod

```diff
@@ -224,5 +222,8 @@ require (
 	gopkg.in/h2non/filetype.v1 v1.0.5 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/minio/madmin-go => ../madmin-go
```

Comment:
You can point it to a fork that will make it compile, eg:
`replace github.com/minio/madmin-go => github.com/klauspost/madmin-go 83c70de1c4cfd39e8caa03bdf311230cbf717078`

Reply:
thanks, learnt something new